src/parser.js

```diff
@@ -164,14 +178,31 @@ const defaultOptions = {
  */
 export function parse(content, baseDir, options = {}) {
     const results = [];
-    const template = options.template || defaultOptions.template;
+    const isTemplateDefault = (options.template == undefined);
+    const isTemplatePath = (templatePath[options.template] == undefined);
+    var tPath;
+    // No template option.
+    if(isTemplateDefault) {
+        tPath = templatePath[defaultOptions.template];
+    }
+    // Template option is a path.
+    else if (isTemplatePath && fs.existsSync(options.template)) {
+        tPath = options.template;
+    }
+    // Template option one of template/ directory.
+    else {
+        tPath = templatePath[options.template] || templatePath[defaultOptions.template];
+    }
+    const template = fs.readFileSync( tPath, "utf-8");
     const unindent = options.unindent || defaultOptions.unindent;
+    const fixlang = options.fixlang || defaultOptions.fixlang;
```

Comment:
`options.fixlang || defaultOptions.fixlang;`

There are confuse case.

`false || true; // => true`

Currently, it is not problem, but it will be problem in the future.

`const fixlang = options.fixlang !== undefined ? options.fixlang : defaultOptions.fixlang;`
is btter.